src/microvis/core/_vis_model.py

```diff
@@ -111,22 +112,29 @@ def backend_adaptor(self, backend: str | None = None) -> AdaptorType:
             self._backend_adaptors[backend] = self._create_adaptor(cls)
         return cast("AdaptorType", self._backend_adaptors[backend])
 
-    def dangerously_get_native_object(self) -> Any:
-        """Return the native object of the backend.
+    @property
+    def backend_adaptors(self) -> Iterable[AdaptorType]:
+        """Convenient, public iterator for backend adaptor objects."""
+        for adaptor in self._backend_adaptors.values():
+            yield adaptor  # type: ignore
```

Comment:
good place for `yield from` :)

side note, if we're certain we _do_ have the proper type here (and this is just something that mypy was never going to be able to determine cause we did the proper type checking in some other context)... then I'd use `cast('AdaptorType'`, adaptor)` here instead of `# type: ignore` (but then `yield from` wouldn't work :smile:)